native/monogame/sdl/MGP_sdl.cpp

```diff
@@ -631,14 +580,77 @@ mgbool MGP_Platform_PollEvent(MGP_Platform* platform, MGP_Event& event_)
             return true;
         }
 
-        case SDL_DROPCOMPLETE:
+        case SDL_EventType::SDL_DROPCOMPLETE:
             event_.Type = MGEventType::DropComplete;
             event_.Drop.Window = MGP_WindowFromId(platform, ev.drop.windowID);
             event_.Drop.File = nullptr;
             return true;
         }
     }
 
+    if(event_.Type == MGEventType::TextEditing || event_.Type == MGEventType::TextInput)
```

Comment:
Maybe move this into a helper method that can be called from both of these events?